rotkehlchen/db/dbhandler.py

```diff
@@ -659,7 +659,11 @@ def remove_from_ignored_assets(self, write_cursor: 'DBCursor', asset: Asset) ->
             (asset.identifier,),
         )
 
-    def get_ignored_assets(self, cursor: 'DBCursor') -> List[Asset]:
+    def get_ignored_assets(self, cursor: 'DBCursor', only_nfts: bool = False) -> List[Asset]:
+        """
+        Retrieve all ignored assets.
+        If `only_nfts` is True, only ignored nfts are returned.
+        """
         cursor.execute(
             'SELECT value FROM multisettings WHERE name="ignored_asset";',
         )
```

Comment:
if you are searching only for nfts maybe is better to add a new query here or extend this. Since you know that nfts identifiers start with `_nft_` you can filter the value in sql and avoid iterating over the whole list of ignored assets

Reply:
yeah yeah.